node/bft/storage-service/src/persistent.rs

```diff
@@ -61,10 +60,10 @@ pub struct BFTPersistentStorage<N: Network> {
 impl<N: Network> BFTPersistentStorage<N> {
     /// Initializes a new BFT persistent storage service.
     pub fn open(storage_mode: StorageMode) -> Result<Self> {
-        let capacity = NonZeroUsize::new(
-            (Committee::<N>::MAX_COMMITTEE_SIZE as usize) * (BatchHeader::<N>::MAX_TRANSMISSIONS_PER_BATCH) * 2,
-        )
-        .ok_or_else(|| anyhow!("Could not construct NonZeroUsize"))?;
+        let max_committee_size = N::LATEST_MAX_CERTIFICATES().unwrap();
```

Comment:
I know these are equivalent but should we be using `Committee::<N>::max_committee_size()` here?

Reply:
Hmm, do we even need `Committee::<N>::max_committee_size()` if this is a `Network`-level value?

Reply:
Here's where we define the function: https://github.com/ProvableHQ/snarkVM/blob/staging/ledger/committee/src/lib.rs#L66. I don't think our consensus is likely to decouple these values soon but I'd suggest using whatever is clearer based on context for now.

Reply:
I don't mind either way; should I change them all? @vicsn I recall having asked in the related snarkVM-side PR if we intend to remove the `Committee` and `BatchHeader` consts in favor of a `Network`-level config, which could be contained in its dedicated `struct`; is this the future plan, or do we still expect to be using config bits from different objects?

Reply:
Let's keep the architecture unchanged - despite being a bit wonky. I agree with Niklas its better for clarity to use the `max_committee_size` where its more directly applicable.